k8s-ci/test

```diff
@@ -34,9 +34,11 @@ function finish {
   # Delete registry images that were used
   deleteRegistryImage "${SECRETLESS_IMAGE_NAME}" "${SECRETLESS_IMAGE_TAG}"
 
-  # Delete test namespace
+  # Delete test namespace for CRD tests and app/backend namespaces for demo test
```

Comment:
This test script is intended to be a generic way to run k8s tests. I think there is stuff left over as part of the conversion from the script only serving the CRD test case.

1. The changeset in this file does cleanup the app/backend namespaces for the demo test. It's doing so by giving the responsibility to the generic script. Ideally this should happen inside the k8s-demo test script. The logic for it does exist inside the k8s-demo test. It's just borked for some reason - https://github.com/cyberark/secretless-broker/blob/master/demos/k8s-demo/stop. 

2. I think we should remove all mentions of CRD in this script.
`SECRETLESS_CRD_SUFFIX` should be passed down to the CRD test script as `TEST_ID`, then we can change that script to receive it as such.

Reply:
Here are the changes I had in mind. https://github.com/cyberark/secretless-broker/compare/test-branch...doodlesbykumbi-patch-1?expand=1